packages/react-components/src/lib/file-picker/file-picker.stories.tsx

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+import React, { useCallback, useState } from 'react';
+import { Story } from '@storybook/react/types-6-0';
+import {
+  FormControl,
+  FormControlLabel,
+  FormLabel,
+  Radio,
+  RadioGroup,
+} from '@material-ui/core';
+import { SupportedLocales } from '../models';
+import { FilePicker } from './file-picker';
+
+export default {
+  title: 'File Picker',
+  component: FilePicker,
+};
+
+export const ReadOnlyMode: Story = () => <FilePicker readOnlyMode={true} />;
+
+export const DarkTheme: Story = () => {
+  return (
+    <FilePicker
+      theme={{
+        primary: 'blue',
+        background: 'black',
+        textOnBackground: 'white',
+        selectionBackground: '#455570',
+      }}
+      isDarkTheme={true}
```

Comment:
Do not think we should expose **isDarkTheme** theme is our own

Reply:
Fixed